modules/python/compat/compat-python-v3.c

```diff
@@ -32,3 +35,22 @@ py_object_as_string(PyObject *object)
 {
   return PyUnicode_AsUTF8(object);
 }
+
+void
+py_setargv(int argc, char **argv)
```

Comment:
This function is... a bit ugh. Not only does it do a whole bunch of things for essentially setting `argv[0]` to syslog-ng (see my note below), but...